metron-stellar/stellar-common/src/main/antlr4/org/apache/metron/stellar/common/generated/Stellar.g4

```diff
@@ -249,6 +253,7 @@ identifier_operand :
   | NULL #NullConst
   | EXISTS LPAREN IDENTIFIER RPAREN #ExistsFunc
   | LPAREN conditional_expr RPAREN #condExpr_paren
```

Comment:
Why are we specifying the #func function?
* We don't implement it
* The name should be better

Reply:
If you specify one named function, you have to name them all.  What would you like to name it?  I didn't spend too much time on it since I knew I wasn't implementing it, admittedly.

Reply:
Oh man, sorry, I did know that from the match work.